src/misc/debug.h

```diff
@@ -19,8 +21,20 @@ extern int debug_level;
 extern _Bool debug_logging;
 
 enum debug_levels { DP_ERROR, DP_WARN, DP_INFO, DP_DEBUG1, DP_DEBUG2, DP_DEBUG3, DP_DEBUG4, DP_TRACE, DP_ALL };
-extern void debug_printf(int level, const char* fmt, ...);
-extern void debug_vprintf(int level, const char* fmt, va_list ap);
```

Comment:
Please preserve these functions as is. The default debug_printf_trace could then just use these.

Reply:
Ok.

Reply:
Ok will do.